README.md

```diff
@@ -1,70 +1,234 @@
+  2 - Select the https tab and copy the url provided. <br></br>
+  3 - Use the git clone command in the root of the folder where you will save the project.<br></br>
+
+Note: For more information you can visit <a href="https://docs.github.com/es/get-started/getting-started-with-git/about-remote-repositories" target="_blank">About remote Repositories</a> and <a href="https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/cloning-and-forking-repositories-from-github-desktop" target="_blank">Adding and cloning repositories</a>.
+
+### Prerequisites
+
+In order to run this project you need:
+- Visual Studio Code
+- Git
+- Npm
+- Node
+- React
+- Jest
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Install
+
+Amongisde the prerequisites, you will need to install the following dependencies: 
+- react-redux
+- redux toolkit
+- jest
+- babel
+- prop-types
+- react-dom
+- react-icons
+- react-router and react-router-dom
+- uuid
+
+All of these dependencies can be installed with [npm](https://www.npmjs.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### Usage <a name="usage"></a>
+
+To run the project, execute the following command in your terminal (inside the project folder):
+
+- npm init
+- npm install
+- npm start
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Run Tests <a name="run-tests"></a>
+
+To run the the tests, you will have to run the following command in the console (inside the project folder): 
+
+- npm install
+
+And then you will have to run the following command in the console (inside the project folder): 
+
+- npm test
+
+You will see the outcome in the console
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+### Deployment <a name="deployment"></a>
+
+To deploy this project in a local enviroment, you will have to run:
+
+- npm start
+
+Then, you will see the project run in a local host: 3000.
+Otherwise, you can see the project deployed in Render in the live demo.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## Author <a name="authors">Vitoesi Elijah</a>
+
+  - GitHub: [@Ellyboi](https://github.com/Ellyboi)
+  - Twitter: [@Elijahvitoesi](https://twitter.com/Elijahvitoesi)
+  - LinkedIn: [Vitoesi Elijah](https://www.linkedin.com/in/vitoesi-elijah-61961611a/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## Future Features <a name="future-features"></a>
+
+
+- [ ] **[Finishing the full website]**
+- [ ] **[Adding more cryptos]**
+- [ ] **[Creating new pages (about, contact, etc)]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- Project Presentation -->
+
+## :movie_camera: Project Presentation <a name="project-presentation"></a>
+
+- [Presentation](Country's Details App - 25 May 2023 - [https://www.loom.com/share/df65b6d1292846c88af2370c5ea6f109]
```

Comment:
- [ ] This is another student presentation video inside your README file. How manage?. Kindly remove this and add your presentation video.

### Wrong Video
![wrong-link](https://github.com/Ellyboi/react-capstone-project/assets/26597432/df6ae57a-c4ab-42f8-9690-47478c931b3a)